docs/advanced/task_management.rst

```diff
@@ -0,0 +1,67 @@
+.. _task_managment:
+
+=================================
+Task Management
+=================================
+.. currentmodule:: qlib
+
+
+Introduction
+=============
+
+The `Workflow <../component/introduction.html>`_ part introduce how to run research workflow in a loosely-coupled way. But it can only execute one ``task`` when you use ``qrun``. To automatically generate and execute different tasks, Task Management module provide a whole process including `Task Generating`_, `Task Storing`_, `Task Running`_ and `Task Collecting`_. 
+With this module, users can run their ``task`` automatically at different periods, in different losses or even by different models.
+
+An example of the entire process is shown `here <>`_.
```

Comment:
Because we don't have real link now, so maybe I need to guess a link based on the naming rules of Github. 
For example:   https://github.com/microsoft/qlib/tree/main/examples/taskmanager/task_manager_rolling_with_updating.py. 
Is that correct?